src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  background-color: $color-white;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  min-width: 900px;
+
+  &__inner {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__menu {
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+  }
+  
+  &__list {
+    list-style-type: none;
+    display: inline-block;
+  }
+  
+  &__link {
+    text-decoration: none;
+    margin-left: 20px;
+    position: relative;
+    display: block;
+    padding: 23px 0;
+    color: #000;
+  }
+  
+  &__link:hover {
+    color: $color-blue;
+  }
+  
+  &__a2 {
+    margin-left: 0;
+  }
+  
+  &__ul {

```

Comment:
Fix this recommendation everywhere
![image](https://user-images.githubusercontent.com/70009205/225949649-3250386d-f770-486d-a5c3-674429936675.png)